docs/content/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "Google Fonts",
+  "url": "https://google-fonts@nuxtjs.org",
```

Comment:
Isn't it https://google-fonts.nuxtjs.org?

Reply:
Oops good catch will fix